streams/se.m3u

```diff
@@ -17,10 +17,8 @@ https://live.tvkomala.com/live/komala/playlist.m3u8
 http://dvrfl05.bozztv.com/gin-komalatv/index.m3u8
 #EXTINF:-1 tvg-id="LifeStyleTV.se@SD",LifeStyleTV (1080p)
 https://app.viloud.tv/hls/channel/711fd3f3a4728449889ee22ba72958ff.m3u8
-#EXTINF:-1 tvg-id="MiracleChannel.se@SD",Miracle Channel (576p)
-https://miracle.stream.boxigy.com/hls/miracle1.m3u8

```

Comment:
No need to remove duplicates, in case if any link won't be resolved in the future, reserve one will take place in generated playlists.
New link can't swich from 360p in VLC, could be just my experience